### ISSUE TITLE
bug: `platform user deleteall` expects a user as argument

### ISSUE THREAD

**Issue by funkyfuture** [mattermost/platform#6925]
#### Summary

`platform user deleteall` expects a user as argument

#### Steps to reproduce

`platform user deleteall`

#### Expected behavior

all users are deleted after a confirmation.

#### Observed behavior (that appears unintentional) 

```
# platform user deleteall --confirm
Error: Enter at least one user.
Usage:
  platform user deleteall [flags]
…
```

#### Version info

```
Version: 4.0.0
Build Number: 4.0.0-rc3
Build Date: Thu Jul 13 10:11:36 UTC 2017
Build Hash: 39154d22aeda926e2af7707352cc65ead0f8d469
Build Enterprise Ready: false
DB Version: 4.0.0
```

**Comment by grundleborg** [comment]
Thanks for the bug report @funkyfuture . I've verified the issue and opened a ticket on our JIRA instance to get it fixed: https://mattermost.atlassian.net/browse/PLT-7145

**Comment by lindy65** [comment]
A PR has been submitted to fix the issue with v4.1 release.
